www/src/templates/component-page-template.tsx

```diff
@@ -75,15 +80,15 @@ export default function PageTemplate({
     };
   }, [components]);
 
-  const cssVariablesTitle = 'Theme Variables (SCSS)';
-  const cssVariablesUrl = 'theme-variables-scss';
+  const scssVariablesTitle = 'Theme Variables (SCSS)';
+  const scssVariablesUrl = 'theme-variables-scss';
 
   const getTocData = () => {
     const tableOfContents = JSON.parse(JSON.stringify(mdx.tableOfContents));
-    if (cssVariables && !tableOfContents.items?.includes()) {
+    if (scssVariablesData && !tableOfContents.items?.includes()) {
```

Comment:
```suggestion
   if (Object.values(scssVariablesData).filter(data => !!data).length) {
```
Since `{}` results always in `true`, so we need to check whether there is any non-empty string data inside each theme.

Reply:
Updated, thanks!

But I think it's better to check this with `Object.values(scssVariablesData).some(data => data)`, which will return `true` if there is at least one non-empty value